sympy/printing/tests/test_latex.py

```diff
@@ -2220,7 +2197,6 @@ def test_MatrixElement_printing():
     F = C[0, 0].subs(C, A - B)
     assert latex(F) == r"\left(A - B\right)_{0, 0}"
 
-    i, j, k = symbols("i j k")
```

Comment:
these are now all integers -- that is probably ok here

Reply:
can this PR be merged